example/build.gradle.kts

```diff
@@ -37,7 +37,7 @@ dependencies {
     implementation(Deps.Kotlin.coroutines)
 
     implementation(Deps.AndroidX.appcompat)
-    implementation(Deps.AndroidX.material)
+    compileOnly(Deps.AndroidX.material)
```

Comment:
why compileOnly?

Reply:
Because is not necessary to include whole dependency into build. Better example in dagger build.gradle.kts